### PR TITLE
Solve all linters, mute print only

### PR DESCRIPTION
**Pull request by maplerichie**
Fixed all linter warnings. #54 
Except `print()`
```dart
include: package:lint/analysis_options.yaml
linter:
  rules:
    avoid_print: false
```

### CODE REVIEW

**Comment by fayaz07** [comment]
did you build the project with xcode? I can see a `Podfile` there

File: analysis_options.yaml [hunk 1]
```diff
@@ -1 +1,4 @@
 include: package:lint/analysis_options.yaml
+linter:
```
Comment (fayaz07) [inline analysis_options.yaml:2]: remove exception for print
and please replace `print` with `debugPrint`

**Review by fayaz07 — APPROVED** [review]

**Review by fayaz07 — CHANGES_REQUESTED** [review]
please remove the print exception and replace them with `debugPrint`

**Comment by maplerichie** [comment]
> did you build the project with xcode? I can see a `Podfile` there

Nope, I code and build on iOS, generated by default.

Will create a new pull request with some changes.

**Comment by fayaz07** [comment]
@maplerichie you could have done those other changes in some other PR

**Review by fayaz07 — APPROVED** [review]

**Comment by maplerichie** [comment]
> @maplerichie you could have done those other changes in some other PR

Discover some minor issues when solving linters, thus solve them along.
Thanks

**Pull request merged**